.github/workflows/tests.yml

```diff
@@ -28,14 +28,88 @@ jobs:
     runs-on: ubuntu-latest
     container: nvidia/cuda:10.2-devel-ubuntu18.04
     steps:
-      - uses: actions/checkout@v1
-        with:
-          submodules: true
-      - name: Add wget
-        run: apt-get update && apt-get install -y wget
-      - name: Install Modern CMake
-        run: wget -qO- "https://cmake.org/files/v3.16/cmake-3.16.0-Linux-x86_64.tar.gz" | tar --strip-components=1 -xz -C /usr/local
-      - name: Configure
-        run: cmake -S . -B build -DCLI11_CUDA_TESTS=ON
-      - name: Build
-        run: cmake --build build
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Add wget
+      run: apt-get update && apt-get install -y wget
+    - name: Install Modern CMake
+      run: wget -qO- "https://cmake.org/files/v3.16/cmake-3.16.0-Linux-x86_64.tar.gz" | tar --strip-components=1 -xz -C /usr/local
+    - name: Configure
+      run: cmake -S . -B build -DCLI11_CUDA_TESTS=ON
+    - name: Build
+      run: cmake --build build -j2
+
+  cmake-config:
```

Comment:
how long does this take to run on all versions of cmake?

Reply:
About 6 seconds each. Plus a little for the initial docker build, total is under 2 minutes.

![Screen Shot 2020-01-08 at 8 31 29 PM](https://user-images.githubusercontent.com/4616906/72030077-f1e2c100-3255-11ea-9217-948f65d6d43a.png)